### ISSUE TITLE
Improve support for customizer

### ISSUE THREAD

**Issue by davejlong** [hominoids/SBC_Case_Builder#7]
Moving thread from https://github.com/hominoids/SBC_Case_Builder/pull/4

Goal of this is to improve the ease of entry for someone who wants to build a case without being familiar with OpenSCAD or needing to figure out the syntax in `sbc_case_builder.cfg`. Make all of the variables populated from `sbc_case_builder.cfg` set by the Customizer. Customizer configurations can be saved as JSON files.

In theory this would also allow for someone to use the [Thingiverse Customizer](https://www.thingiverse.com/app:22) to get a case without having to download the SCAD files at all.

Current hangup is how to handle the accessory settings.

**Comment by hominoids** [comment]
After doing a deeper dive into the latest features of the customizer I don’t believe a fully integrated solution using the json format is going to possible.  We need to be able to search the accessories and  search() doesn't appear to work with that format. It only works on vectors and lists of vectors.  The json entries are also sorted alphabetically which rules out any fixed reference scheme.  I did have a idea for a solution but it is a hybrid with a philosophical change.

The current approach has been to treat a complete case as one entry.  By decoupling the accessories from the case, the customizer can be used to create and store the case information.  The current sbc_case_builder.cfg would still be used for the accessory entries in the current format.  Still looking for other ways but I did put some time into the fore mentioned possible approach for discussion.   Take a look at the case_design branch I just posted and let me know what you think.  Didn't remove the case data yet from sbc_case_builder.cfg but it's not used.
